src/components/FolderContent.js

```diff
@@ -0,0 +1,14 @@
+import '../styles/FolderContent.css';
+import SearchBar from '../components/SearchBar';
+import CardList from '../components/CardList';
+
+const FolderContent = ({ folderInfo }) => {
+  return (
+    <div className="folder-content">
+      <SearchBar />
+      <CardList folderInfo={folderInfo} />
+    </div>
```

Comment:
header,footer잘 하셨는데 `main`이 없네요. 
```suggestion
    <main className="folder-content">
      <SearchBar />
      <CardList folderInfo={folderInfo} />
    </main>
```

Reply:
2e2f79993e4c3c3f2de4c227fd637bdf5912d8a7 `<main>` 추가했습니다!